test/integration/main_test.go

```diff
@@ -135,6 +135,11 @@ func GithubActionRunner() bool {
 	return os.Getenv("GITHUB_ACTIONS") == "true"
 }
 
+// Arm64Platform returns true if running on arm64/* platform
+func Arm64Platform() bool {
```

Comment:
I feel like this helper can be removed. It saves zero lines of code and abstracts away an important distinction.

Reply:
For me this helper is a kind of "bookmark function". It's used only in integration tests and helps to find all places where we have exceptions for arm64.

Reply:
fixed